src/llm/apis/openai_completions.cpp

```diff
@@ -109,15 +217,27 @@ absl::Status OpenAIChatCompletionsHandler::parseChatCompletionsPart() {
     if (request.logprobschat && request.stream) {
         return absl::InvalidArgumentError("logprobs are not supported in streaming mode.");
     }
-
+    // max_completion_tokens: uint; optional
+    it = doc.FindMember("max_completion_tokens");
```

Comment:
shouldn't it be in parseCommonPart?

does it differ from "max_tokens" processing? if not, we should reorganize it to do not duplicate the code and its value validation

Reply:
max_completion_tokens refers only to chat completions api, not to the completions api